import-service/services/import-service.ts

```diff
@@ -1,37 +1,47 @@
 import { StorageService } from './storage-service';
 import { Readable } from 'stream';
 import * as csvParser from 'csv-parser';
+import { SQS } from 'aws-sdk';
 
 export class ImportService {
   private BUCKET_NAME = process.env.PRODUCT_BUCKET_NAME;
+  private QUEUE_URL = process.env.QUEUE_URL;
   private UPLOADED_FOLDER_PREFIX = 'uploaded/';
   private storageService: StorageService;
+  private sqs: SQS;
 
   constructor() {
     this.storageService = new StorageService();
+    this.sqs = new SQS();
   }
 
   getSignedUrlPromise(fileName: string): Promise<string> {
-    const key = `${this.UPLOADED_FOLDER_PREFIX}${fileName}`;
+    const key = `${ this.UPLOADED_FOLDER_PREFIX }${ fileName }`;
     const contentType = 'text/csv';
 
     return this.storageService.getSignedUrlPromise(this.BUCKET_NAME, key, contentType);
   };
 
   parseFiles(keys: string[]): Promise<void[]> {
     const streamsArray = keys.map(async (key) => {
-      const stream = this.storageService.getObject(this.BUCKET_NAME, key).createReadStream();
-      return this.parseCSVFile(stream);
+      const stream = await this.storageService.getObject(this.BUCKET_NAME, key);
```

Comment:
you don't need to await getObject. Just call createReadStream

Reply:
yea I was trying to fix issue which I wrote about in description, however this doesn't help